content/scalers/mysql.md

```diff
@@ -0,0 +1,101 @@
++++
+fragment = "content"
+weight = 100
+title = "MySQL"
+background = "light"
++++
+
+Scale applications based on MySQL query result.
+
+**Availability:** v1.1+ | **Maintainer:** Community
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `mysql` trigger that scales based on result of MySQL query.
+The trigger requires the following information:
+- `query` - a MySQL query that should return single numeric value that will be used for HPA
```

Comment:
HPA is implementation detail, not everybody will be aware what it is since we abstract that out.
```suggestion
- `query` - a MySQL query that should return single numeric value that will be used for scaling
```

Alternatively, we can even shorten it more
```suggestion
- `query` - a MySQL query that should return single numeric value
```